packages/financial-templates-lib/src/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -287,8 +287,10 @@ class CryptoWatchPriceFeed extends PriceFeedInterface {
       })
       .flat();
     const startTime = endTime - this.twapLength;
+    const twapPrice = computeTWAP(priceTimes, startTime, endTime, this.web3.utils.toBN("0"));
 
-    return computeTWAP(priceTimes, startTime, endTime, this.web3.utils.toBN("0"));
+    if (!this.invertPrice) return twapPrice;
```

Comment:
Prob could make this a one liner: `return (this.invertPrice ? this._invertPriceSafely(twapPrice) : twapPrice)`

Reply:
Yeah good point. Updated to this.